docs/synapse/userguides/model_updates/update_v2_193_0.rst

```diff
@@ -0,0 +1,227 @@
+
+  ``id``
+    The ID parsed from the "message-id" header.
+
+
+``inet:flow``
+  The form had the following property added to it:
+
+  ``capture:host``
+    The host which captured the flow.
+
+
+``inet:service:account``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the account.
+
+
+``inet:service:bucket``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the bucket.
+
+
+``inet:service:bucket:item``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the bucket item.
+
+
+``inet:service:channel``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the channel.
+
+
+``inet:service:channel:member``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the channel membership.
+
+
+``inet:service:emote``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the emote.
+
+
+``inet:service:group``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the group.
+
+
+``inet:service:group:member``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the group membership.
+
+
+``inet:service:permission``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the permission.
+
+
+``inet:service:relationship``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the relationship.
+
+
+``inet:service:rule``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the rule.
+
+
+``inet:service:session``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the session.
+
+
+``inet:service:subscription``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the subscription.
+
+
+``inet:service:tenant``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the tenant.
+
+
+``inet:service:thread``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the thread.
+
+
+``it:dev:repo:issue:label``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the object.
+
+
+``it:host``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the object.
+
+
+``it:host:tenancy``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the object.
+
+
+``it:software:image``
+  The form had the following property added to it:
+
+  ``url``
+    The primary URL associated with the object.
```

Comment:
Are these missing a template value for something more specific than "the object"?

Reply:
`object` is the default value for `inet:service:object`. Do we want to change that for the various objects that are inheriting from it?

Reply:
Seems like the elements inheriting from it should override it, i tossed up a PR to add them here https://github.com/vertexproject/synapse/pull/4054